pkg/skaffold/deploy/helm/parse.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helm
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes/scheme"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/types"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/manifest"
+	"github.com/sirupsen/logrus"
+)
+
+func parseReleaseInfo(namespace string, b *bufio.Reader) []types.Artifact {
+	var results []types.Artifact
+
+	r := k8syaml.NewYAMLReader(b)
+	for i := 0; ; i++ {
```

Comment:
nit: for { ...}

Reply:
wouldn't mind that change, but this code isn't new so i'll leave it untouched for now